packages/evolution-frontend/src/components/inputs/InputMapFindPlace.tsx

```diff
@@ -155,6 +156,7 @@ export class InputMapFindPlace extends React.Component<
         /*if (!this.props.value) { // does not work because map is not yet loaded at this point
             this.geocodePlaces(bbox);
         }*/
+       this.renderGeocodeButton = true;
```

Comment:
Small typo. It should be renderGeoCodeButton.

Reply:
Can you please add a brief comment beforehand to explain what will happen? Right now it's clear to me, but in 1 year, we might have forgotten.